### ISSUE TITLE
Trouble installing a new agent on a manual installation

### ISSUE THREAD

**Issue by Zonalhaz** [huginn/huginn#2553]
Hi,
Background: I have a huginn instance running on an ubuntu aws lightsail box. I followed the [instructions for manual install](https://github.com/huginn/huginn/blob/master/doc/manual/installation.md).

I wanted to install a new agent: the [todoist agent](https://github.com/stesie/huginn_todoist_agent) but have been having some problems and was hoping for a bit of help if possible. I've followed the instructions on that github (which are to add `huginn_todoist_agent` to `ADDITIONAL_GEMS=` in the .env file and then run bundle.

Running `sudo -u huginn -H bundle` results in the following error:
```
You are trying to install in deployment mode after changing
your Gemfile. Run 'bundle install' elsewhere and add the
updated Gemfile.lock to version control.

If this is a development machine, remove the /home/huginn/huginn/Gemfile freeze
by running 'bundle install --no-deployment'.

The dependencies in your gemfile changed

You have added to the Gemfile:
* huginn_todoist_agent
```
Running `sudo -u huginn -H bundle install` results in the same error.

I'm not too familiar with bundle & gems in general so I'm likely making some fundamental mistake, but I can't seem google my way out of this.
Any help would be greatly appreciated.

Thanks!

**Comment by dsander** [comment]
Hi,

sorry for the late reply. That is indeed a situation we did not think about. I think running bundler this way should work:

```
sudo -u huginn -H bundle install --no-deployment --without development test
```

**Comment by Zonalhaz** [comment]
Hi,
That gets me through that error, thanks!
I've ended up having to use "`sudo bundle install --no-deployment --without development test`" as I was getting git errors (see below) that I was having a hard time fixing. 

I'm a bit worried of possible issues that may occur 'cause of omitting the "`-u huginn -H`" but I have huginn running and with the todoist agents installed so thank you!

**Git problems**
This is straying out of huginn a little bit but I'm now getting the error: "`error: cannot open .git/FETCH_HEAD: Permission denied`" from running the command when including "`-u huginn -H`".
Looks like a permission error from some googling but as far as I can tell the huginn user owns that file and the two parent folders. "ls -l" on the file gives: "`-rw-rw-r--  1 huginn huginn  3707 Jun 29 09:53 FETCH_HEAD`" so I'm not sure what's up with that.

Either way I have accomplished the thing I set out to do so thank you!
